pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/OfflineClusterIntegrationTest.java

```diff
@@ -427,6 +428,21 @@ public void testUploadSegmentRefreshOnly()
     cleanupTestTableDataManager(offlineTableName);
   }
 
+  @Test
+  public void testReloadStatusApi() {
+    String reloadJobId = reloadOfflineTableAndValidateResponse(getTableName(), false);
+    if (reloadJobId != null) {
+      // Spin until validated
+      TestUtils.waitForCondition(aVoid -> {
+        try {
+          return validateReloadJobSuccess(reloadJobId);
```

Comment:
What I meant is that we can add the `validateReloadJobSuccess()` after every `reloadOfflineTableAndValidateResponse()` in other methods. All the segments should be successfully reloaded after the `TestUtils.waitForCondition()` check is passed

Reply:
Ack.